src/string_immutable.zig

```diff
@@ -1227,12 +1227,38 @@ pub fn utf16Codepoint(comptime Type: type, input: Type) UTF16Replacement {
     }
 }
 
+pub fn convertUTF16ToUTF8(list_: std.ArrayList(u8), comptime Type: type, utf16: Type) !std.ArrayList(u8) {
+    var list = list_;
+    const length = bun.simdutf.length.utf8.from.utf16.le(utf16);
```

Comment:
This gets the length twice

It should only do it once.

Reply:
I suggest adding a version of this function which doesn't check the length and then having a default version call this function & pass the length in